packages/generator-one-app-module/generators/app/templates/intl-child-module/src/components/ModuleContainer.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { loadLanguagePack, updateLocale } from '@americanexpress/one-app-ducks';
+import { FormattedMessage, IntlProvider } from 'react-intl';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { holocronModule } from 'holocron';
+import { fromJS } from 'immutable';
+
+export const <%=moduleNamePascal%> = ({ switchLanguage, languageData, localeName }) => {
+  const locales = ['en-US', 'en-CA', 'es-MX'];
+  // naive solution - up to user on how to load in data
```

Comment:
I think this comment should be rephrased a bit. Not sure what but something about this comment doesn't sound right to me. Same comment applies to the other templates.

Reply:
what about 
```
  // short-term fix to prevent render errors - for loading async data, read: 
  // https://github.com/americanexpress/one-app/blob/master/docs/api/modules/Loading-Data.md
```

or just 
```
// Read about loading async data: 
// https://github.com/americanexpress/one-app/blob/master/docs/api/modules/Loading-Data.md
```

Reply:
fixed per in-person convo.